test/release/examples/users-guide/base/configs2.compopts-fake

```diff
@@ -0,0 +1,8 @@
+chpl -sverbose=true configs2.chpl
+chpl -sverbose configs2.chpl
+chpl -sbitMask=0b00001111 -ssize=16 -sage=uint(16) configs2.chpl
+chpl -sage=real(32) configs2.chpl
```

Comment:
I think we should also use quotes around command-line flags that include parens, e.g. -sage=uint(16)

Otherwise some readers (like myself) will ponder how this is supposed to work with bash/tcsh/...